OrbitAccessibility/CMakeLists.txt

```diff
@@ -0,0 +1,45 @@
+# Copyright (c) 2020 The Orbit Authors. All rights reserved.
+# Use of this source code is governed by a BSD-style license that can be
+# found in the LICENSE file.
+
+cmake_minimum_required(VERSION 3.15)
+
+project(OrbitAccessibility CXX)
+add_library(OrbitAccessibility STATIC)
+
+target_compile_options(OrbitAccessibility PRIVATE ${STRICT_COMPILE_FLAGS})
+
+target_sources(
+  OrbitAccessibility
+  PUBLIC AccessibleInterface.h
+         AccessibleInterfaceRegistry.h
+         AccessibleWidgetBridge.h
+         )
```

Comment:
`PUBLIC` headers should be in the subdirectory `include/OrbitAccessibility` directory. Follow the example of other modules that do this.

Reply:
Thanks for pointing this out, I was following the wrong example for a new module - I've changed this and all the includes accordingly.